metals/src/main/scala/scala/meta/internal/metals/codeactions/MillifyScalaCliDependencyCodeAction.scala

```diff
@@ -0,0 +1,116 @@
+package scala.meta.internal.metals.codeactions
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+import scala.meta.internal.metals.Buffers
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.internal.metals.codeactions.CodeAction
+import scala.meta.internal.metals.codeactions.MillifyScalaCliDependencyCodeAction._
+import scala.meta.internal.parsing.Trees
+import scala.meta.io.AbsolutePath
+import scala.meta.pc.CancelToken
+import scala.meta.tokens.Token._
+
+import org.eclipse.{lsp4j => l}
+
+class MillifyScalaCliDependencyCodeAction(buffers: Buffers) extends CodeAction {
+
+  override def kind: String = l.CodeActionKind.QuickFix
+
+  override def contribute(params: l.CodeActionParams, token: CancelToken)(
+      implicit ec: ExecutionContext
+  ): Future[Seq[l.CodeAction]] = Future {
+
+    val path = params.getTextDocument().getUri().toAbsolutePath
+    val range = params.getRange()
+    val couldBeScalaCli = path.isScalaScript || path.isScala
+
+    val tokenized =
+      if (couldBeScalaCli && range.getStart == range.getEnd)
+        for {
+          buffer <- buffers.get(path)
+          line = buffer.linesIterator
+            .drop(range.getStart.getLine)
+            .take(range.getEnd().getLine() - range.getStart().getLine() + 1)
+            .mkString("\n")
```

Comment:
Does `range.getStart == range.getEnd` indicate that we could do just `take(1)`? Seems so, as what we want to do is "drop lines before cursor, take that one only"

Reply:
Ach yes, unless we would want to convert multiple lines at the same time in that case we could drop the condition.

Though we would have to merge the resulting text edits, so maybe it's better to just switch to take(1)

Reply:
> You need to run sbt scalafixAll to fix the scalafix job. I tried pushing to your branch, but I think you disabled it.

Done, I was looking for something like `prePR` initially :wink: